### ISSUE TITLE
Sound static

### ISSUE THREAD

**Issue by totopesce** [07th-mod/umineko-answer#36]
When some voices are playing you can hear some static, when the voice ends the problem ends.
If its of any help in the umineko project changin the audioformat to f32 fixed this issue, ont knowif its possible to do something similar on this.

Thanks

**Comment by drojf** [comment]
Funnily enough, I think I was the one who originally raised that issue on the forums with Umineko Project, ...but that issue was a problem specific to their special game engine. We use the POnscripter engine which shouldn't have this problem.

Can you give an example (post audio recording of the problem)? Also, what output format is your system set to use (in case it makes a difference):

![image](https://user-images.githubusercontent.com/1249449/35837785-27bb8214-0b3c-11e8-851a-b5216979850b.png)

Also, the only thing we can do is re-encode the ps3 game's audio files to a different format. We cannot adjust anything in the engine like Umineko Project did.

**Comment by totopesce** [comment]
I have the same configuration in playback devices as you showed in the image.
havent been able to reproduce it, its kind of random, doesnt happen everytime even on the same seen, but it sound kind of like this, but less noticeable, and some distortion on the bgm.
https://mega.nz/#!p5JEAQAD!DUhQa7impQvKJRFJ4hNP7kxwCednDpmMalrrHtreV70

greetings

**Comment by drojf** [comment]
Do you know if it's clipping and not static? If that's the case, you can try to adjust the music volume in the script file until the clipping goes away

1. Open the 0.utf file in notepad++ or some other decent text editor
2. search for `default_vol` . You should find the following text:

```
numalias default_vol,70
;If the music and sound effects are too loud/quiet, they can be adjusted by tweaking musicVol (used to be called volX)
;Volume is multiplied by musicVol/volY, both must be integers. Default is 50/100
numalias musicVol 50
numalias volY 100
;voicevol must be an integer in the range 0-100
voicevol 100
```

3. Reduce musicVol, voicevol, and default_vol by some amount (for example I would try default_vol = 35, musicVol = 25, voicevol = 50). Then set your system volume higher to compensate.

Let me know if that makes any difference or if it's a different problem entirely.

**Comment by totopesce** [comment]
it seems it was clipping did what you said and it stopped, sounded similar to the problem on umineko project.

Thanks

**Comment by drojf** [comment]
Thanks for reporting the issue - I will have a play around with the default values to figure out some defaults which prevent clipping from happening.

**Comment by drojf** [comment]
Now that we added the voice/music adjustment feature to both games, I think this is fixed: (see commit b5104935592344dc8f2388ccc472e21be031908e)
